snap/snapcraft.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+---
```

Comment:
Did `yamllint` require you to declare a new document here? Generally you shouldn't include this since `snapcraft.yaml` is a one-document YAML file.

Reply:
Yup, `yamllint` complained:
`snap/snapcraft.yaml
  3:1       warning  missing document start "---"  (document-start)`